test/src/test/java/test/org/dst/core/KVStoreTest.java

```diff
@@ -33,9 +29,50 @@ public void testList() {
     list.add("v2");
     list.add("v3");
     ks.list().put("k1", list);
+
+    //get
     Assertions.assertEquals(list, ks.list().get("k1"));
+    //del
     Assertions.assertTrue(ks.list().del("k1"));
     Assertions.assertNull(ks.list().get("k1"));
+
+    //lput
+    List<String> list1 = new ArrayList<String>();
```

Comment:
Could you please separate this test to many small cases, like `testLput()`, `testRput()`, etc.

Reply:
ha-ha. I was going to split up.
But I saw him before, and I thought you were asking for it.
I'll fix it